HomeAssistant/Resources/en.lproj/Localizable.strings

```diff
@@ -445,6 +445,8 @@
 "actions_configurator.rows.text.title" = "Text";
 "actions_configurator.rows.text_color.title" = "Text Color";
 "actions_configurator.title" = "New Action";
+"actions_configurator.trigger_example.title" = "Example Trigger";
+"actions_configurator.trigger_example.share" = "Share Contents";
```

Comment:
I wordsmithed this a few times and didn't land on anything much better than "Share Contents".